ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/InteractiveStatementExecutor.java

```diff
@@ -117,8 +123,8 @@ void handleStatement(final QueuedCommand queuedCommand) {
     throwIfNotConfigured();
 
     handleStatementWithTerminatedQueries(
-        queuedCommand.getCommand(),
```

Comment:
nit: I'd just pass command, commandId, status, and offset in as parameters rather than deserializing from this method